src/tike/view.py

```diff
@@ -58,11 +58,83 @@
 import matplotlib.transforms as transforms
 from matplotlib import collections
 import matplotlib.pyplot as plt
+import matplotlib.colors as mplcolors
+import cv2 as cv
 import numpy as np
 import tike.linalg
 
 logger = logging.getLogger(__name__)
 
+def complexHSV_to_RGB( img0 ):
+    """Convert a complex valued array to RGB representation.
+    
+    Takes a complex valued array, represents the phase as hue,
+    magnitude as value, and saturation as all ones in a new (..., 3) shaped
+    array. This is then converted to the RGB colorspace.
+
+    Parameters
+    ----------
+    img0 : :py:class:`numpy.array`
+        A (...) shaped complex64 numpy array.
+
+    Returns
+    -------
+    rgb_img : :py:class:`numpy.array`
+        The (..., 3) shaped array which represents the input complex valued array 
+        in a RGB colorspace.
+    """
+    
+    sz      = img0.shape
+
+    hsv_img = np.ones( ( *sz, 3 ), 'float32' )
+
+    # Rescale hue and value to the range [0, 1].
+    hsv_img[ ..., 0 ] = np.angle( img0 ) 
+    hsv_img[ ..., 0 ] -= np.min( hsv_img[ ..., 0 ] )
+    hsv_img[ ..., 0 ] /= np.max( hsv_img[ ..., 0 ] )
+
+    hsv_img[ ..., 2 ] = np.abs( img0 )
+    hsv_img[ ..., 2 ] -= np.min( hsv_img[ ..., 2 ] )
+    hsv_img[ ..., 2 ] /= np.max( hsv_img[ ..., 2 ] )
+
+    rgb_img = mplcolors.hsv_to_rgb( hsv_img )
+
+    return rgb_img
+    
+
+def resize_complex_image( img0, scale_factor = ( 1, 1 ), interpolation = cv.INTER_LINEAR ):
+    """Takes a M0 x N0 complex valued array, splits it up into real and imaginary,
+    and resizes (interpolates) the horizontal and vertical dimensions, yielding 
+    a new array of size M1 x N1. The result can then be  used for further plotting using 
+    e.g. imshow() or imsave() from matplotlib.
+
+    Parameters
+    ----------
+    img0 : :py:class:`numpy.array`
+        A M0 x N0 complex64 numpy array.
+    scale_factor : 2 element positive valued float tuple, 
+        ( vertical resize/scale, horizontal resize/scale  )
+    interpolation  : int 
+        cv.INTER_NEAREST  = 0, cv.INTER_LINEAR = 1
+        cv.INTER_CUBIC    = 2, cv.INTER_AREA   = 3
+        cv.INTER_LANCZOS4 = 4 
+
+    Returns
+    -------
+    imgRS : :py:class:`numpy.array`
+        The new M1 x N1 which has been resized according to the scale factors above.
+    """
+    
+    dim  = ( int( img0.shape[1] * scale_factor[1] ), 
+             int( img0.shape[0] * scale_factor[0] ) )
```

Comment:
```suggestion
    dim  = ( int( img0.shape[0] * scale_factor[0] ), 
             int( img0.shape[1] * scale_factor[1] ) )
```
I think these should be reversed? Not sure. We want all axes to be consistently ordered. i.e. If we have a `(2, 2)` shaped array and give `scale_factor=(2, 5)`, the output shape should be `(4, 10)` not `(10, 4)`.